package.json

```diff
@@ -22,7 +23,7 @@
     "tsc": "ethereumjs-config-tsc",
     "test": "npm run test:node && npm run test:browser",
     "test:browser": "npm run build && karma start karma.conf.js",
-    "test:node": "npm run build && tape test/*.js | tap-prettify -"
+    "test:node": "npm run build && tape -r ts-node/register test/*.ts | tap-prettify -"
```

Comment:
Just gave the tests a manual local run, looks good, test number is preserved (127 tests). 😋